crates/node/src/main.rs

```diff
@@ -93,10 +93,14 @@ async fn main() -> anyhow::Result<()> {
     );
 
     let l1_gas_setter = GasPriceProvider::new();
+
+    if let Some(fix_gas) = run_cmd.l1_sync_params.gas_price {
+        l1_gas_setter.update_eth_l1_gas_price(fix_gas as u128);
+        l1_gas_setter.update_eth_l1_data_gas_price(fix_gas as u128);
```

Comment:
should we have two different flags for these in the CLI?

Reply:
resolved